.github/workflows/website.yml

```diff
@@ -6,6 +6,7 @@ on:
       - main
```

Comment:
I bit late, but we should also allow branches `1.x`, `2.x` and `3.x` to trigger rebuild.

Reply:
I think it should work because we run this work flow when a new release is pushed and the older versions will create it